.github/workflows/builds.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           ver=($(lxd-snapcraft -get-version -file ~/lxd-imagebuilder-pkg-snap-lp/snapcraft.yaml))
           rsync -a --exclude .git --delete . ~/lxd-imagebuilder-pkg-snap-lp/
           cd ~/lxd-imagebuilder-pkg-snap-lp
-          lxd-snapcraft -set-version "${ver[0]}" -set-source-commit "${ver[1]}"
+          lxd-snapcraft -package lxd-imagebuilder -set-version "${ver[0]}" -set-source-commit "${ver[1]}"
```

Comment:
We should make that arg mendatory, so that even in the `lxd` case we provide it. This way, search and replace will catch those.

Reply:
Go for it